src/C0_data_processing.py

```diff
@@ -313,12 +319,15 @@ def energyProduction(dict_values, group):
         )
 
         if FILENAME in dict_values[group][asset]:
-            receive_timeseries_from_csv(
-                dict_values,
-                dict_values[SIMULATION_SETTINGS],
-                dict_values[group][asset],
-                "input",
-            )
+            if dict_values[group][asset][FILENAME] in ["None", None]:
```

Comment:
```suggestion
            if dict_values[group][asset][FILENAME] in ("None", None):
```
To remind ourselves that tuples also exist and should be preferred in case of immutable variables